content/learn/book/migration-guides/0.5-0.6/_index.md

```diff
@@ -29,9 +29,9 @@ members = [ "my_crate1", "my_crate2" ]
 
 ### "AppBuilder" was merged into "App"
 
-All functions of {{rust_type(type="struct" crate="bevy_app" mod="" version="0.5.0" name="AppBuilder" no_mod=true)}} were merged into {{rust_type(type="struct" crate="bevy_app" mod="" version="0.6.0" name="App" no_mod=true)}}.
+All functions of [`AppBuilder`] were merged into [`App`].
 
-In practice this means that you start constructing an {{rust_type(type="struct" crate="bevy_app" mod="" version="0.6.0" name="App" no_mod=true)}} by calling {{rust_type(type="struct" crate="bevy_app" mod="" version="0.6.0" name="App" no_mod=true method="new")}} instead of {{rust_type(type="struct" crate="bevy_app" mod="" version="0.5.0" name="App" no_mod=true method="build")}} and {{rust_type(type="trait" crate="bevy_app" mod="" version="0.5.0" name="Plugin" no_mod=true method="build")}} takes a {{rust_type(type="struct" crate="bevy_app" mod="" version="0.6.0" name="App" no_mod=true)}} instead of a {{rust_type(type="struct" crate="bevy_app" mod="" version="0.5.0" name="AppBuilder" no_mod=true)}}
+In practice this means that you start constructing an [`App`] by calling `App::new()` instead of `App::build` and `Plugin::build` takes a [`App`] instead of a [`AppBuilder`].
```

Comment:
These functions used to be links. Can you re-add those? (and do a pass to make sure we haven't dropped links elsewhere? I'm seeing this happen in a number of places.)